crates/ruff_linter/src/settings/mod.rs

```diff
@@ -129,6 +131,9 @@ macro_rules! display_settings {
     (@field $fmt:ident, $prefix:ident, $settings:ident.$field:ident | quoted) => {
         writeln!($fmt, "{}{} = \"{}\"", $prefix, stringify!($field), $settings.$field)?;
     };
+    (@field $fmt:ident, $prefix:ident, $settings:ident.$field:ident | globmatcher) => {
```

Comment:
Nit: I think this fits better with the naming scheme used elsewhere in the macro.
```suggestion
    (@field $fmt:ident, $prefix:ident, $settings:ident.$field:ident | glob) => {
```

Reply:
Happy to change it, but does it shift your preference at all to note that globset has both `GlobMatcher` and `Glob` as distinct types, and this added case only works for a setting that is a `GlobMatcher`? For a `Glob` you can just use the regular default display with good results.

Reply:
Okay, I can see why using the full type name makes sense. I'm not too opinionated on the naming scheme (it's not that consistent to begin with, anyway) so we can keep using that. Thanks for explaining 👍